arduino/smartcar/smartcar.ino

```diff
@@ -14,18 +18,40 @@ void setup() {
 }
 
 void loop() {
-  car.setSpeed(90);
+  car.setSpeed(70); 
   preventCrash();
 }
 
-void preventCrash() {
-  unsigned int fDistance = front.getDistance();
-  if (fDistance > 0 && fDistance < 200) {
+void preventCrash(){
+  preventCrashWithUltrasonic(); 
+  preventCrashWithInfrared(frontIR, 0, 150); 
+  preventCrashWithInfrared(leftIR, 50, 150);
+  preventCrashWithInfrared(rightIR, 50, 150);
+  preventCrashWithInfrared(backIR, 50, 150);
```

Comment:
I don't think the infrared sensors can measure that far away. At least not the real ones.
Have you verified these are valid min and max distances?

Reply:
The infrared sensors are in the range between 0 cm and 400 cm.

Reply:
> The infrared sensors are in the range between 0 cm and 400 cm.

Nope. According to the [documentation](https://github.com/ItJustWorksTM/smce-gd/wiki/Vehicle-Capabilities#infrared-distance-sensor):

> Maximum distance of .4 meters, no minimum distance.